package.json

```diff
@@ -9,6 +9,10 @@
   "directories": {
     "test": "test"
   },
+  "preferGlobal": true,
```

Comment:
remove this - it's deprecated. https://docs.npmjs.com/files/package.json#preferglobal